docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -156,3 +156,25 @@ struct S
     public static void M([UnscopedRef] ref int x) { }
 }
 ```
+
+## `Microsoft.CodeAnalysis.EmbeddedAttribute` is validated on declaration
+
+***Introduced in Visual Studio 2022 version 17.13***
+
+The compiler now validates the shape of `Microsoft.CodeAnalysis.EmbeddedAttribute` when declared in source. Previously, the compiler
```

Comment:
Unrelated to the PR: it's not clear to me what the `Microsoft.CodeAnalysis.EmbeddedAttribute` does. Is that documented somewhere?

Reply:
Not documented afaik, though we will when we add the SG API. It prevents the compiler from resolving the type outside the current compilation. It's how the compiler itself gets around the "type defined in multiple assemblies" issues when it needs to synthesize an attribute.